ci/env.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
-set -e
+
```

Comment:
Why have we got this if [ ! -z $assets_dir ] bit twice ?

Reply:
merge issue. thanks